### PR TITLE
Add MIDI Keyboard documentation

### PR DESCRIPTION
**Pull request by Jacalz**
## Description
Follow up to #180 now that we have `qsynth` in the repository. This makes it a lot easier for users to connect their keyboards on Solus. There is also a section about Keyboards that are known to work since that is generally quite hard to find when checking keyboards. As far as I know there isn't really any good lists of Keyboards that work on Linux, my keyboard doesn't have driver support on Linux by M-audio but it works anyhow.

Any feedback is appreciated... 

### Submitter Checklist

- [x] Updated the "lastmod" portion at the top of any modified Markdown files.
- [x] Squashed commits with `git rebase -i` (if needed)

### CODE REVIEW

File: hardware/midi-keyboard/en.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+lastmod = "2019-03-20T19:45:00+01:00"
++++
+# Using USB connected MIDI Keyboards
+
+These instructions will guide you through how to set up and use an USB-connected MIDI Keyboard on Solus. This will easily let you use the keyboard to play music with a wide set of instruments.
+
+## Installing required software
+
+First we'll need to install `qsynth` and `qjackctl` using the following command:
+```bash
```
Comment (JoshStrobl) [inline hardware/midi-keyboard/en.md]: Add newline above, space before bash.

File: hardware/midi-keyboard/en.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+sudo eopkg it qsynth qjackctl
+```
+
+Then we want to install a soundfont in order to get sounds and instruments that play when you press the keys on the keyboard.
+```bash
```
Comment (JoshStrobl) [inline hardware/midi-keyboard/en.md]: Add newline above, space before bash.

File: hardware/midi-keyboard/en.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
++++
+title = "Using USB connected MIDI Keyboards"
+lastmod = "2019-03-20T19:45:00+01:00"
++++
+# Using USB connected MIDI Keyboards
+
+These instructions will guide you through how to set up and use an USB-connected MIDI Keyboard on Solus. This will easily let you use the keyboard to play music with a wide set of instruments.
```
Comment (JoshStrobl) [inline hardware/midi-keyboard/en.md]: *a USB-connected

File: hardware/midi-keyboard/en.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+{{< altimg "qsynth-soundfont.jpg" "help-center/hardware/midi-keyboard/" >}}
+
+## Connecting the keyboard
+
+At this stage you need to first verify that the keyboard is plugged in and turned on. Now we just need to open up `qjackctl` to connect the keyboard on a software level. In `qjackctl` you press on `Graph` and drag the keyboard's output to the input called `FLUID Synth` (Your keyboard name will probably differ from the on in the image below). This step will need to be done every time you disconnect or turn off you keyboard.
+
+{{< altimg "qjackctl-connect.jpg" "help-center/hardware/midi-keyboard/" >}}
+
+You should now be ready to play on the keyboard using `qsynth`.
+
```
Comment (JoshStrobl) [inline hardware/midi-keyboard/en.md]: Fix double newline.

File: hardware/midi-keyboard/en.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+Then we want to install a soundfont in order to get sounds and instruments that play when you press the keys on the keyboard.
+```bash
+mkdir -p $HOME/soundfonts
+wget http://http.debian.net/debian/pool/main/f/fluid-soundfont/fluid-soundfont_3.1.orig.tar.gz -O $HOME/soundfonts/fluid-soundfont.tar.gz
+cd $HOME/soundfonts && tar xvf fluid-soundfont.tar.gz fluid-soundfont-3.1/FluidR3_GM.sf2 --strip=1
+rm fluid-soundfont.tar.gz
+```
+
+## Setting up the soudfonts in qsynth
```
Comment (JoshStrobl) [inline hardware/midi-keyboard/en.md]: soundfonts

File: hardware/midi-keyboard/en.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+## Connecting the keyboard
+
+At this stage you need to first verify that the keyboard is plugged in and turned on. Now we just need to open up `qjackctl` to connect the keyboard on a software level. In `qjackctl` you press on `Graph` and drag the keyboard's output to the input called `FLUID Synth` (Your keyboard name will probably differ from the on in the image below). This step will need to be done every time you disconnect or turn off you keyboard.
+
+{{< altimg "qjackctl-connect.jpg" "help-center/hardware/midi-keyboard/" >}}
+
+You should now be ready to play on the keyboard using `qsynth`.
+
+
+# Keyboard compatibility
```
Comment (JoshStrobl) [inline hardware/midi-keyboard/en.md]: Should be header level 2

File: hardware/midi-keyboard/en.md [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+
+The following keyboards have been tested and/or suggested to function correctly by our users.
+
+This list should not suggest that *only* such devices listed below are compatible with Solus, as there may be devices not listed below that are in fact compatible.
+
+## Models
```
Comment (JoshStrobl) [inline hardware/midi-keyboard/en.md]: Unnecessary header, already obvious we're talking about various models / MIDI keyboards.

**Comment by Jacalz** [comment]
All comments have been addressed by latest commit :slightly_smiling_face:

**Comment by JoshStrobl** [comment]
Thanks for the documentation,  I'll get it deployed as soon as I'm available to do so :+1:

**Comment by Jacalz** [comment]
> Thanks for the documentation, I'll get it deployed as soon as I'm available to do so 👍

You are very welcome, but you’re on vacation as far as I know  so there’s no need to hurry 😉

**Pull request merged**